prototype/link/scripts/retinotopy/calculate_surface_volume.sh

```diff
@@ -3,20 +3,21 @@
 # Take a freesurfer directory, a hemisphere, and a niml file from a participants ROI tracing and create text files that have the volume at each node (for wm and pial separately).
 # This can then be used by $PROJ_DIR/scripts/retinotopy/Retinotopy.ipynb to calculate the volume per ROI and plot the results
 # For instance you can use these commands:
-# ./scripts/retinotopy/calculate_surface_volume.sh iBEAT rh rh.manual
+# ./scripts/retinotopy/calculate_surface_volume.sh analysis/freesurfer/iBEAT/SUMA/ iBEAT rh rh.manual 1
 #
 #SBATCH --output=./logs/calculate_surface_volume-%j.out
 #SBATCH -p short
 #SBATCH -t 4:00:00
 #SBATCH --mem 20000
 
 # Get the inputs
-FREESURFER_NAME=$1 # What is the name of the freesurfer directory being loaded. Must be in analysis/freesurfer/
-hemisphere=$2 # What hemisphere are you using
-niml_name=$3 # What is the prefix of the ROI you saved
+SUMA_folder=$1 # What SUMA folder do you want to load? almost certainly analysis/freesurfer/$FREESURFER_NAME/SUMA/, unless running from the project directory
+FREESURFER_NAME=$2 # What is the name of the freesurfer directory being loaded. Must be in 
```

Comment:
Added. Also, to comment on above: I have hopefully clarified what wasn't clear, although I didn't make much change to the checkerboard figure since the code explains what is going on for interested parties. The question asks why the ROIs were broken down differently for the mappers but I don't think that is true.